tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/nullable/NullableTypeTest.java

```diff
@@ -78,6 +79,22 @@ public void testNilableTypeArrayIteration() {
         validateError(result, 0, "incompatible types: expected '(()|any)', found '(()|any)?'", 33, 19);
     }
 
+    @Test(dataProvider = "dataToTestNullInJSONRelatedContext", description = "Test null in JSON related context")
+    public void testNullInJSONRelatedContext(String functionName) {
+        BRunUtil.invoke(result, functionName);
+    }
+
+    @DataProvider
+    public Object[] dataToTestNullInJSONRelatedContext() {
```

Comment:
We don't need to say "JSON context" anymore, right?

Reply:
Ah my bad! missed this. Will fix :)